### PR TITLE
Add descriptions for DCR datasets.

### PR DESCRIPTION
**Pull request by isullivan**
These changes will be merged to master after DM-9615 has been merged.

### CODE REVIEW

File: policy/datasets.yaml [hunk 1]
```diff
@@ -581,6 +588,9 @@ dcrCoadd_forced_config:
     python: lsst.meas.base.forcedPhotCoadd.ForcedPhotCoaddConfig
     template: config/forcedPhotCoadd.py
 dcrCoadd_forced_src:
+    description: >
+        Catalog of forced measurements (shape and position parameters held fixed) on the dcrCoadd.
+        Use dcrCoadd_ref as the reference catalog by default.
```
Comment (parejkoj) [inline policy/datasets.yaml]: "Uses"?

Comment (parejkoj) [reply policy/datasets.yaml]: Include what Task produces it?

File: policy/datasets.yaml [hunk 2]
```diff
@@ -606,6 +620,9 @@ dcrCoadd_det_schema:
     python: lsst.afw.table.SourceCatalog
     template: schema/dcrCoadd_det.fits
 dcrCoadd_mergeDet:
+    description: >
```
Comment (parejkoj) [inline policy/datasets.yaml]: Is it too late to object to this name and rename it `dcrCoadd_mergeDetections`?

Comment (isullivan) [reply policy/datasets.yaml]: I think so. The *Coadd_mergeDet name is used in many other places, and I'm just following the existing pattern.

**Comment by isullivan** [comment]
Replaced by pull request #103

**Pull request closed without merging**